flow/Makefile

```diff
@@ -162,6 +162,11 @@ export PLACE_PINS_ARGS
 
 export FLOW_VARIANT ?= base
 
+# Env variable for DREAMPlace
+export DREAM_PLACE ?= 0
+export PYTHON_CMD ?= python
+export DREAM_PLACE_INSTALL ?= xx/DREAMPlace/install/
```

Comment:
`DREAM_PLACE_INSTALL` should use the same behaviour of other tools, e.g., https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/8d1989f94a95cfa39bd77c3b538a285689654215/flow/Makefile#L244-L247

Reply:
DREAM Place is launched using a python file (which is available in the DREAM Place installation directory). I am not sure how to replicate the same for DREAMPlace.

Reply:
You can use something like:
```
export DREAM_PLACE_INSTALL ?= $(abspath $(FLOW_HOME)/../tools/DREAMPlace
export DREAM_PLACE_EXE ?= $(DREAM_PLACE_INSTALL)/<python_script_name>
export DREAM_PLACE_ARGS ? = if they are needed
```
Replace `<python_script_name>` with the correct name.